db/schema.sql

```diff
@@ -2,8 +2,8 @@
 -- PostgreSQL database dump
 --
 
--- Dumped from database version 11.5
```

Comment:
just wanted to double check - do we want to commit this schema change in this PR?

Reply:
I think so, but happy to take it out if folks disagree. My rationale was that our docker-compose script is using pg 12 and the schema in that repo is as well